source/material_model/utilities.cc

```diff
@@ -699,6 +699,43 @@ namespace aspect
       }
 
 
+      // Calculate a parameter of single composition at single point by averaging phases.
+      // Either linear(by_log is 0) or using log form(by_log is 1), default is linear.
```

Comment:
Remove the comment above. Documentation about the general purpose of this function belongs in the header file.